source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "rake db:schema:load vs rake db:migrate"
+date: 2016-03-23 08:09:25 +0100
```

Comment:
@Ikera before merging to master and publishing make sure you update date/time to today and rename the file too.